src/ImageSharp/MetaData/Profiles/Exif/ExifProfile.cs

```diff
@@ -92,21 +93,17 @@ public ExifProfile(ExifProfile other)
         /// <summary>
         /// Gets or sets which parts will be written when the profile is added to an image.
         /// </summary>
-        public ExifParts Parts
-        {
-            get;
-            set;
-        }
+        public ExifParts Parts { get; set; }
 
         /// <summary>
         /// Gets the tags that where found but contained an invalid value.
         /// </summary>
-        public IEnumerable<ExifTag> InvalidTags => this.invalidTags;
+        public IReadOnlyList<ExifTag> InvalidTags => this.invalidTags;
```

Comment:
@dlemstra are you happy with `IReadOnlyList`? It seems fine to me, because it lets indexing the collection while keeping it immutable.